src/owl/owl_database.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+"""Configuration definition."""
+
+__all__ = ["OwlDatabase"]
+
+import asyncio
+import typing
+
+import aiopg
+import aiopg.sa
+import structlog
+
+from owl.create_messages_table import create_messages_table
+
+
+class OwlDatabase:
+    """Connection to the OWL database and message table.
+
+    Parameters
+    ----------
+    url
+        URL of OWL database server in the form:
+        postgresql://[user[:password]@][netloc][:port][/dbname]
+    table
+        Name of table holding OWL messages.
+    """
+
+    def __init__(self, url: str, table: str = "owl_messages"):
+        self._closed = False
+        self.url = url
+        self.table = table
+        self.connection = None
+        self.logger = structlog.get_logger("OwlDatabase")
+        self.engine = None
+        self.table = create_messages_table(create_indices=False)
```

Comment:
It creates a sqlalchemy table which is used to query a database. It can also be used to create the table, but only by suitable calls to sqlalchemy. I will expand the function doc string to explain this.